src/components/announcements/IntroProjectTags.tsx

```diff
@@ -0,0 +1,10 @@
+export default function IntroProjectTags() {
```

Comment:
Is it worth wrapping these in trans?

Reply:
oh yes, they definitely should be. these are placeholders for now but will wrap them before launch